packages/snap/src/index.tsx

```diff
@@ -401,8 +419,14 @@ export const processRpcRequest: SnapRpcProcessor = async (
     case RpcMethods.GetZkCertStorageHashes: {
       // This method only returns a single hash of the storage state. It can be used to detect changes, for example if the user imported another zkCert in the meantime.
       // Because it does not leak any personal or tracking data, we do not ask for confirmation.
+
+      const chainID =
+        (request?.params as GetZkCertStorageHashesRequest)?.chainID ?? 843843;
```

Comment:
I think it would make more sense to not filter for the chainID if there is no chainID parameter in the query.